src/main/java/net/snowflake/ingest/streaming/internal/ParquetTypeGenerator.java

```diff
@@ -1,16 +1,19 @@
 /*
- * Copyright (c) 2022 Snowflake Computing Inc. All rights reserved.
+ * Copyright (c) 2022-2024 Snowflake Computing Inc. All rights reserved.
  */
 
 package net.snowflake.ingest.streaming.internal;
 
+import static net.snowflake.ingest.utils.IcebergDataTypeParser.deserializeIcebergType;
```

Comment:
nit: don't think this import static adds much value, it'd be preferable to read `IcebergDataTypeParser.deserializeIcebergType` at the callsite instead of just `deserializeIcebergType`.

Reply:
Removed and call with class name as prefix.